### DISCUSSION TITLE
Eclispe connection

### DISCUSSION THREAD

**Discussion by Garret-Chmielewski** [community/community#68756] [New to GitHub]
I try and connect my github account with my IED Eclipse and its not accepting it. I dont know how to fix it

**Comment by sreshtech** [comment]
To connect your GitHub account with Eclipse IDE, start by ensuring that you have the EGit plugin installed. EGit is responsible for Git integration within Eclipse. You can add EGit to your Eclipse IDE by going to "Help" > "Eclipse Marketplace" and searching for "EGit."

Once EGit is installed, configure your GitHub account settings within Eclipse. Go to "Window" > "Preferences," expand the "Team" section, select "Git," and click "Configuration." Here, set your Git user information, ensuring your name and email match your GitHub account.
